thehive4py/endpoints/observable.py

```diff
@@ -111,3 +111,13 @@ def count(self, filters: Optional[FilterExpr] = None) -> int:
             params={"name": "observable.count"},
             json={"query": query},
         )
+
+    def download_attachment(
+        self, observable_id: str, attachment_id: str, attachment_path: str, as_zip: bool = False
```

Comment:
I'd rename the `attachment_path` argument to `observable_path` to better align with the existing naming conventions in the create methods.

Btw did you test it against a hive instance to verify that the method works? I tried on mine and I received some weird errors, trying to upgrade to the latest version to see if it's only because of the outdated API.

Reply:
Alright will change the variable name. 
I only tried it on my thehive instance (v5). I did not try it on v3 and v4.

Reply:
variable has been rename in [4f2a0a3](https://github.com/TheHive-Project/TheHive4py/pull/266/commits/4f2a0a37ae7d5e4d823b9af73856d6b477b1d9b9)